reactDashboard/src/components/GeoHeatMap/FullscreenControl.css

```diff
@@ -1,5 +1,30 @@
+.full-screen {
+  padding-top: 10px;
+}
+
+.darkmode .fullscreen {
+  color: #121212;
+  background-color: #c1def1;
+}
+
 .fullscreen svg {
-    height: 25px;
-    width: 25px;
-    color: #319bd5;
+  height: 25px;
+  width: 25px;
+  color: #133a53;
+}
+
+.fullscreen:hover,
+.fullscreen svg:hover {
+  color: white;
+  background-color: #133a53;
+}
+
+.darkmode .fullscreen:hover {
+  border-color: 1px solid #c1def1;
```

Comment:
Ok - we need a 
    border: 1px solid whateverColour
in a block .fullscreen:hover, and 
    border-color: #c1def1 
here

Reply:
no longer required - refactored